lib/db/mongo.ts

```diff
@@ -0,0 +1,7 @@
+import mongoose from "mongoose";
+
+export async function connectToMongo() {
+  if (!process.env.MONGODB_URI) throw new Error("MONGODB_URI not set");
```

Comment:
I've added an env.example file to list the required environment variables. Since this is a smaller project, I opted to keep it simple without a dedicated config.ts file, but yes can consider adding it if the project scales or if it would improve maintainability.